<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>智能电梯广告系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QWidget" name="elevatorWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_elevator">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_elevator">
         <property name="title">
          <string>电梯状态</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_status">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <widget class="QWidget" name="floorDisplayWidget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_floor">
             <property name="spacing">
              <number>10</number>
             </property>
             <item>
              <spacer name="horizontalSpacer_left">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_floor_number">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>200</height>
                </size>
               </property>
               <property name="text">
                <string>1</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_floor_arrow">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>60</height>
                </size>
               </property>
               <property name="text">
                <string>⏸</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_direction">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>60</height>
                </size>
               </property>
               <property name="text">
                <string>停止</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_right">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_buttons">
         <property name="title">
          <string>选择楼层</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_buttons">
          <property name="spacing">
           <number>10</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="btn_floor_8">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>8楼</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="btn_floor_7">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>7楼</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="btn_floor_6">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>6楼</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="btn_floor_5">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>5楼</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="btn_floor_4">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>4楼</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="btn_floor_3">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>3楼</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="btn_floor_2">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>2楼</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="btn_floor_1">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>1楼</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="adWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_ad">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_ad">
         <property name="title">
          <string>广告播放</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_ad_content">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <widget class="QStackedWidget" name="stackedWidget_ad">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="page_image">
             <layout class="QVBoxLayout" name="verticalLayout_image">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_ad_image">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>300</height>
                 </size>
                </property>
                <property name="text">
                 <string>等待加载广告...</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_text">
             <layout class="QVBoxLayout" name="verticalLayout_text">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_ad_text">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>300</height>
                 </size>
                </property>
                <property name="text">
                 <string>暂无文字广告</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_ad_controls" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_ad_controls">
             <property name="spacing">
              <number>10</number>
             </property>
             <item>
              <spacer name="horizontalSpacer_ad_left">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btn_ad_prev">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>35</height>
                </size>
               </property>
               <property name="text">
                <string>上一个</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_ad_next">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>35</height>
                </size>
               </property>
               <property name="text">
                <string>下一个</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_ad_pause">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>35</height>
                </size>
               </property>
               <property name="text">
                <string>暂停</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_ad_resume">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>35</height>
                </size>
               </property>
               <property name="text">
                <string>继续</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_ad_right">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>